### ISSUE TITLE
[Feature Request] Edit EPUB metadata

### ISSUE THREAD

**Issue by crisalis2** [foobnix/LibreraReader#613]
It would be nice if you could edit basic metadata inside Librera, like title and author. Many books have wrong or unwanted metadata. I would like to be able to fix it directly inside Librera instead of having to use some other program like Calibre.

**Comment by alittlebitofit** [comment]
any update?

**Comment by amunizp** [comment]
Just saw that the file in the case of PDF seems to have the link to update metadata under file info.
![Screenshot_20230702-010425_1](https://github.com/foobnix/LibreraReader/assets/5783911/b8787629-0e22-45a3-8e2c-88fc0efc33a3)

**Comment by alittlebitofit** [comment]
@amunizp It closes that popup when I click on it. It doesn't work me, does it work for you?

**Comment by amunizp** [comment]
No. So let's just say it is not related to pdf or ePub. I just saw that the main developer is in a warzone. This is probably something that is going to be implemented. Related: #1126
